src/models/task.js

```diff
@@ -0,0 +1,3 @@
+const tasks=[];
```

Comment:
what are you trying to do with this file?

Reply:
tasks [], I tried to use it in another folder as a placeholder for our data. like a database